docs/platforms/javascript/guides/aws-lambda/profiling/index.mdx

```diff
@@ -49,14 +49,15 @@ Sentry.init({
   profilesSampleRate: 1.0,
 });
 
+// Profiling happens automatically after setting it up with `Sentry.init()`.
```

Comment:
I think this is a little confusing for me because profiling requires the existence of transactions/spans, so it only starts automatically if `Sentry.init` creates transactions via automatic instrumentation. Could we make that more clear?

Reply:
Do you have a suggestion of how to write this a little bit more understandable?

Reply:
I think we can keep this line if we use the suggestion from @vivianyentran in https://github.com/getsentry/sentry-docs/pull/9783#discussion_r1576628367 because of the `All spans captured in your application will have profiling data attached to them` line!